### PR TITLE
Water Cup

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+.cup {
+  width: 200px;
+  height: 220px;
+
+  border: 6px solid #262626;
+  border-top: 2px solid transparent;
+  border-radius: 15px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
```
Comment (OSAMA478) [inline style.css]:
```suggestion
	border-radius: 5px 5px 15px 15px;
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  box-shadow: 0 0 0 6px #fff, 0 20px 30px rgba(0, 0, 0, 1);
+  background: url(https://image.ibb.co/fmHm66/wave.png);
+
```
Comment (OSAMA478) [inline style.css:24]:
```suggestion
	background-size: cover;
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  0% {
+    background-position: 0 0;
+  }
+  25% {
+    background-position: 0 80px;
+  }
+  100% {
+    background-position: 500px -40px;
+  }
+}
```
Comment (OSAMA478) [inline style.css]:
```suggestion
@keyframes animate {
	0% {
		background-position: 0 150px;
	}
	10% {
		background-position: 84px 150px;
	}
	20% {
		background-position: 168px 112.5px;
	}
	30% {
		background-position: 252px 75px;
	}
	40% {
		background-position: 336px 37.5px;
	}
	50% {
		background-position: 420px 0px;
	}
	60% {
		background-position: 504px 37.5px;
	}
	70% {
		background-position: 588px 75px;
	}
	80% {
		background-position: 672px 112.5px;
	}
	90% {
		background-position: 756px 150px;
	}
	100% {
		background-position: 845px 150px;
	}
}
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  border-radius: 15px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+
+  box-shadow: 0 0 0 6px #fff, 0 20px 30px rgba(0, 0, 0, 1);
+  background: url(https://image.ibb.co/fmHm66/wave.png);
+
+  background-repeat: repeat-x;
+
+  animation: animate 10s linear infinite;
```
Comment (OSAMA478) [inline style.css:28]:
```suggestion
  animation: animate 5s linear infinite;
```

**Comment by OSAMA478** [comment]
i think above changes will improve your **animation**....

**Review by anask601 — APPROVED** [review]

**Review by mmusk — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css]: Changes made as per comment. thanks

File: style.css (hunk 2 above)
Comment (nehalskdev) [reply style.css:24]: Changes made as per comment. thanks

File: style.css (hunk 3 above)
Comment (nehalskdev) [reply style.css]: Changes made as per comment. thanks

File: style.css (hunk 4 above)
Comment (nehalskdev) [reply style.css:28]: i think 10s makes it better

**Pull request merged**